vod/hls/m3u8_builder.c

```diff
@@ -511,48 +511,55 @@ m3u8_builder_build_master_playlist(
 	media_set_t* media_set,
 	vod_str_t* result)
 {
-	media_track_t** cur_sequence_tracks;
-	media_sequence_t* cur_sequence;
-	media_track_t* track;
+	media_sequence_t* cur_sequence;	
 	media_info_t* video;
 	media_info_t* audio = NULL;
 	uint32_t sequence_index;
 	uint32_t bitrate;
 	u_char* p;
-	size_t max_video_stream_inf;
-	size_t result_size;
+	size_t max_video_stream_inf = 0;
```

Comment:
no need to initialize max_video_stream_inf, result_size, audio_track (after removing the if that i requested below)